src/components/shared/SearchInput.js

```diff
@@ -39,6 +41,27 @@ const SearchInput = React.createClass({
     return Svg("magnifying-glass");
   },
 
+  renderNav() {
+    const { count, handleNext, handlePrev } = this.props;
+    if (!count || count == 1) {
+      return;
+    }
+
+    return dom.div(
+      { className: "search-nav-buttons" },
+      Svg("arrow-down", {
+        className: classnames("nav-btn", "next"),
+        onClick: handleNext,
+        title: "Next Result",
+      }),
+      Svg("arrow-up", {
+        className: classnames("nav-btn", "prev"),
+        onClick: handlePrev,
+        title: "Previous Result",
+      })
+    );
+  },
```

Comment:
SearchInput could be a good component to snapshot test:

https://facebook.github.io/jest/docs/snapshot-testing.html#content